### ISSUE TITLE
Unable to submit a new post

### ISSUE THREAD

**Issue by jorenbroekema** [plaintextoffenders/plaintextoffenders#2]
http://plaintextoffenders.com/submit doesn't seem to work for me, it's just blank. 

Anywhere, here's my submission.

Domain: http://www.altijdwonen.nl/ 
What they do: They're a "professional" real estate company in the Netherlands
Their offense: storing password/username in plain text, emailing them on signup. 

Evidence: 
Signup screen -> https://puu.sh/AvN1n/a1a14afe3f.png so basically you don't even get to set your own password and they create an ID for you. Assuming the ID started at 1 and auto-increments, there's now 42870 users on this site of which passwords are stored in plain text. (because my ID is 42870). 
Email with ID + password: https://puu.sh/AvN50/572e62cc43.png
Cherry on top of the cake: They don't use SSL / HTTPS at all. 

Kind regards,
Joren

**Comment by lmcmenemie** [comment]
Due to some badly implemented GDPR consent plugin from tumblr, the forms on the website are not showing. Surprised nothing has been done about it.

**Comment by guysoft** [comment]
Hey, is this still being maintained, submitted a new page a few months ago (cpnshop.co.il), but I can't see it went up here.

**Comment by guyshe-jfrog** [comment]
Looks like its dead - anyone know if there is an alternative?

**Comment by RokeJulianLockhart** [comment]
> [`#issuecomment-1941296131`](https://github.com/plaintextoffenders/plaintextoffenders/issues/2#issuecomment-1941296131)

@guyshe-jfrog, I just submit new submissions as issues prefixed with “[Submission]”:

1. [`issues/23`](https://github.com/plaintextoffenders/plaintextoffenders/issues/23#issue-2060427570)

1. [`issues/24`](https://github.com/plaintextoffenders/plaintextoffenders/issues/24#issue-2143555048)

1. [`issues/27`](https://github.com/plaintextoffenders/plaintextoffenders/issues/27#issue-3787007708)

**Comment by guysoft** [comment]
@RokeJulianLockhart Those are stuck on open though. Did anyone actually go over it and close it for you?

**Comment by RokeJulianLockhart** [comment]
@guysoft, nope, but they're at least permanent and discussable. Better than not reporting them, since I can periodically collate some into [`pull/28`](https://github.com/plaintextoffenders/plaintextoffenders/pull/28#issue-3787025021).
